src/commcare_cloud/commands/secrets.py

```diff
@@ -47,7 +47,13 @@ def _secrets_edit(self, environment, secret_name):
         try:
             secret_value = json.loads(secret_value)
         except ValueError:
-            pass
+            old_value = environment.secrets_backend.get_secret(secret_name)
+            if isinstance(old_value, list) or isinstance(old_value, dict):
+                puts(color_error(
+                    'Could not understand value, please use valid json: {value}'
+                    .format(value=secret_value)
```

Comment:
It seems bad or at least inconsistent to print the secret value after accepting it using `getpass` (concealed input).